src/en/help-oracle.md

```diff
@@ -100,138 +100,157 @@ Cloud Types
 
-Enter a name, a CIDR formatted address for the `IP Address Prefix`, and an
-optional description with one or more tags. Use the `IP Exchange` drop-down
-menu to select the exchange created previously and click on `Create`. 
+![Create an IP network](./media/oracle_create-ip-network-2.png)
 
-A few moments later, the new network will be listed.
+Enter a name, a CIDR formatted address for the 'IP Address Prefix', and an
+optional description with one or more tags. Use the 'IP Exchange' drop-down
+menu to select the exchange created previously and click on 'Create'. 
 
-When you next bootstrap Juju with Oracle Compute, you'll be able to use these
+When you next create a Juju controller (see below), you'll be able to use these
 new subnets and spaces. For example, typing `juju subnets` will show output
 similar to the following:
 
 ```no-highlight
 subnets:
-  10.0.123.0/24:
+  192.168.0.0/16:
     type: ipv4
-    provider-id: /Compute-a476989/juju-network
+    provider-id: /Compute-a498151/peter.gargamel@example.com/oracle-cloud-ip-network
     status: in-use
-    space: juju-exchange
+    space: oracle-cloud-ip-exchange
     zones:
     - default
 ```
 
-Typing `juju spaces` will list the Oracle IP exchange:
+Typing `juju spaces` will list the exchange:
 
 ```no-highlight
-Space          Subnets
-juju-exchange  10.0.123.0/24
+Space                     Subnets
+oracle-cloud-ip-exchange  192.168.0.0/16
 ```
 
 See [How to configure more complex networks using spaces][spaces] for further
 details on networks and spaces. 
 
-## Create controller
+## Create the Juju controller
 
-To bootstrap your Oracle Compute cloud, use the `juju bootstrap` command with
-your cloud name:
+You are now ready to create a Juju controller:
 
 ```bash
-juju boostrap myoracle
+juju bootstrap oracle-cloud oracle-cloud-controller
 ```
 
-You can now start deploying Juju charms and bundles to your Oracle cloud.
+Above, the name given to the new controller was 'oracle-cloud-controller'.
+Oracle Compute will create an instance to run the controller on.
+
+Once created, you can view the controller as an Oracle Compute instance by
+navigating to 'My Services URL', opening the left menu (top-left icon), and
+selecting 'Compute'. The controller should be visible under the 'Instances'
+tab:
+
+![List controller instance](./media/oracle_bootstrap-instances-2.png)
 
-A successful bootstrap and deployment will result in the controller environment
-being visible in the `Instances` page of the Oracle Compute dashboard:
+Oracle Compute has the notion of *sites* (like availability zone or region).
+You may need to change your site to see your new controller. This is done in
+the top-right corner:
+
+![Oracle Compute sites](./media/oracle_bootstrap-instances-sites.png)
+
+## Next steps
+
+You can now start deploying Juju charms and/or bundles to your Oracle cloud.
+Continue with Juju by visiting the [Models][models] and
+[Introduction to Juju Charms][charms] pages.
```

Comment:
Right. I left a similar section in the [MAAS review](https://github.com/juju/docs/pull/1971/commits/4554b5628e72243132089d164e7dd7f28a80e818). What do you think?